code/modules/client/preference_setup/general/02_body.dm

```diff
@@ -65,24 +49,42 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
 	pref.preview_icon = null
 	from_file(S["bgstate"], pref.bgstate)
 
+	// Grandfathering in older saves post colour rewrite.
```

Comment:
I'll note that `/datum/category_item/player_setup_item` datums have a  `/update_setup()`-proc just for this sort of thing